sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample_AnalyzeWithPrebuiltModel.md

```diff
@@ -0,0 +1,260 @@
+                {
+                    IReadOnlyDictionary<string, DocumentField> itemFields = itemField.AsDictionary();
+
+                    if (itemFields.TryGetValue("Description", out DocumentField itemDescriptionField))
+                    {
+                        if (itemDescriptionField.ValueType == DocumentFieldType.String)
+                        {
+                            string itemDescription = itemDescriptionField.AsString();
+
+                            Console.WriteLine($"  Description: '{itemDescription}', with confidence {itemDescriptionField.Confidence}");
+                        }
+                    }
+
+                    if (itemFields.TryGetValue("Amount", out DocumentField itemAmountField))
+                    {
+                        if (itemAmountField.ValueType == DocumentFieldType.Double)
+                        {
+                            double itemAmount = itemAmountField.AsDouble();
+
+                            Console.WriteLine($"  Amount: '{itemAmount}', with confidence {itemAmountField.Confidence}");
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    if (document.Fields.TryGetValue("SubTotal", out DocumentField subTotalField))
+    {
+        if (subTotalField.ValueType == DocumentFieldType.Double)
+        {
+            double subTotal = subTotalField.AsDouble();
+            Console.WriteLine($"Sub Total: '{subTotal}', with confidence {subTotalField.Confidence}");
+        }
+    }
+
+    if (document.Fields.TryGetValue("TotalTax", out DocumentField totalTaxField))
+    {
+        if (totalTaxField.ValueType == DocumentFieldType.Double)
+        {
+            double totalTax = totalTaxField.AsDouble();
+            Console.WriteLine($"Total Tax: '{totalTax}', with confidence {totalTaxField.Confidence}");
+        }
+    }
+
+    if (document.Fields.TryGetValue("InvoiceTotal", out DocumentField invoiceTotalField))
+    {
+        if (invoiceTotalField.ValueType == DocumentFieldType.Double)
+        {
+            double invoiceTotal = invoiceTotalField.AsDouble();
+            Console.WriteLine($"Invoice Total: '{invoiceTotal}', with confidence {invoiceTotalField.Confidence}");
+        }
+    }
+}
+```
+
+To see the full example source files, see:
+
+* [Analyze with prebuilt model from URI](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample_AnalyzeWithPrebuiltModelFromUriAsync.cs)
+* [Analyze with prebuilt model from file](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample_AnalyzeWithPrebuiltModelFromFileAsync.cs)
+
+[businessCard_fields]: https://aka.ms/formrecognizer/businesscardfields
+[idDocument_fields]: https://aka.ms/formrecognizer/iddocumentfields
+[invoice_fieds]: https://aka.ms/formrecognizer/invoicefields
+[receipt_fields]: https://aka.ms/formrecognizer/receiptfields
```

Comment:
we will need to update these links once the new ones are created. Cata is creating them.
this shouldn't block your PR though. we can always update them later